apps/studio/prisma/schema.prisma

```diff
@@ -221,3 +222,33 @@ model Whitelist {
 
   @@index([email])
 }
+
+enum AuditLogEvent {
+  ResourceCreate
+  ResourceUpdate
+  ResourceDelete
+  ResourceMove
+  UserCreate
+  UserUpdate
+  UserDelete
+  Publish
+  Login
+  Logout
+  PermissionCreate
+  PermissionUpdate
+  PermissionDelete
+}
+
+model AuditLog {
+  id        Int           @id @default(autoincrement())
+  user      User          @relation(fields: [userId], references: [id])
+  userId    String
+  eventType AuditLogEvent
+  createdAt DateTime      @default(now())
+  updatedAt DateTime      @default(now()) @updatedAt
+  metadata  Json
+  /// @kyselyType(PrismaJson.AuditLogJsonContent)
+  /// [AuditLogJsonContent]
+  delta     Json
```

Comment:
nit: hmm I got a bit confused by the type naming, maybe `AuditLogJsonDeltaContent`? I thought the original name refers to the `metadata` field.

Reply:
sounds good

Reply:
for consistency, calling this `AuditLogDeltaJsonContent`